test_deepdoc_env.py

```
@@ -0,0 +1,105 @@
+        ('deepdoc.vision.table_structure_recognizer', 'TableStructureRecognizer'),
+        ('deepdoc.vision.recognizer', 'Recognizer'),
+        ('deepdoc.parser.pdf_parser', 'RAGFlowPdfParser'),
+        ('deepdoc.parser.docx_parser', 'RAGFlowDocxParser'),
+    ]
+
+    for module_name, class_name in modules:
+        try:
+            module = __import__(module_name, fromlist=[class_name])
+            getattr(module, class_name)
+            results.append((module_name, class_name, True, None))
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)